src/NUnitFramework/nunitlite.runner/nunitlite.runner-netcore/Runner/AutoRun.cs

```diff
@@ -0,0 +1,86 @@
+﻿// ***********************************************************************
```

Comment:
A couple of these runner classes were getting really complicated and hard to follow with the defines for all the different platforms, so it was easier with a couple to make copies, clean them up and rewrite them just for DNX.